ui/react-components/components/AddAppointment/AddAppointment.jsx

```diff
@@ -222,45 +222,68 @@ const AddAppointment = props => {
         setShowSuccessPopup(true);
     };
 
+    const setServiceErrorMessageFromResponse = response => {
+        response.error && response.error.message ? setServiceErrorMessage(response.error.message)
```

Comment:
does it need response null check?
response && response.error && ...?

Reply:
@AlekhyaYalla We are doing this check where ever we are calling the **setServiceErrorMessageFromResponse** . so its not required here

Reply:
okay